hubspot.com.hubspot-template-3.json

```diff
@@ -0,0 +1,22 @@
+{
+    "providerId": "hubspot.com",
+    "providerName": "HubSpot",
+    "serviceId": "hubspot-template-3",
+    "serviceName": "HubSpot CMS Site",
+    "version": 1,
+    "logoUrl": "https://domainconnect.org/wp-content/uploads/2017/02/HubSpot-768x223.png",
+    "description": "Enables an apex domain to work with the HubSpot CMS",
+    "variableDescription": "IPs: HubSpot CMS IPs",
+    "hostRequired": true,
```

Comment:
Why is hostRequired set to true?  Usually this is required when the template doesn't work with an apex domain (e.g. you have a cname record using the fqdn, and the template would only work with a host/sub-domain).

Reply:
Good catch, I will remove it